src/tests/ftest/pool/create_all_hw.py

```diff
@@ -0,0 +1,319 @@
+                for smd_device in nvme_device["smd_devices"]:
+                    smd_vos_bytes = min(smd_vos_bytes, smd_device["avail_bytes"])
+                    smd_vos_size += host_size
+
+        self.log.info("Available VOS size: scm_bytes=%d, scm_size=%d, smd_bytes=%d, smd_size=%d",
+                scm_vos_bytes, scm_vos_size, smd_vos_bytes, smd_vos_size)
+
+        scm_pool_bytes = scm_vos_bytes * scm_vos_size if scm_vos_bytes != sys.maxsize else 0
+        smd_pool_bytes = smd_vos_bytes * smd_vos_size if smd_vos_bytes != sys.maxsize else 0
+        self.log.info("Available POOL size: scm_bytes=%d, smd_bytes=%d",
+                scm_pool_bytes, smd_pool_bytes)
+
+        return scm_pool_bytes, smd_pool_bytes
+
+    def test_one_pool(self):
+        """Test the creation of one pool with all the storage capacity
+
+        Test Description:
+            Create a pool with all the capacity of all servers. Verify that the pool created
+            effectively used all the available storage and there is no more available storage.
+
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,large
+        :avocado: tags=pool,pool_create_all
+        :avocado: tags=pool_create_all_one_hw
+        """
+        self.log.info("Test  basic pool creation with full storage")
+
+        self.log.info("Creating a pool with 100% of the available storage")
+        self.add_pool_qty(1, namespace="/run/pool/*", create=False)
+        self.pool[0].size.update("100%")
+        self.pool[0].create()
+        self.assertEqual(self.pool[0].dmg.result.exit_status, 0,
+                "Pool could not be created")
+
+        self.log.info("Checking size of the pool")
+        self.pool[0].get_info()
+        tier_bytes = self.pool[0].info.pi_space.ps_space.s_total
+        self.assertLessEqual(abs(self.scm_avail_bytes - tier_bytes[0]), self.delta_bytes,
+                "Invalid SCM size: want={}, got={}, delta={}".format(self.scm_avail_bytes,
+                    tier_bytes[0], self.delta_bytes))
+        self.assertLessEqual(abs(self.smd_avail_bytes - tier_bytes[1]), self.delta_bytes,
+                "Invalid SMD size: want={}, got={}, delta={}".format(self.smd_avail_bytes,
+                    tier_bytes[1], self.delta_bytes))
+
+        self.log.info("Checking size of available storage")
+        tier_bytes = self.get_pool_available_bytes()
+        self.assertEqual(0, tier_bytes[0],
+                "Invalid SCM size: want=0, got={}".format(tier_bytes[0]))
+        self.assertEqual(0, tier_bytes[1],
+                "Invalid SMD size: want=0, got={}".format(tier_bytes[1]))
+
+    def test_recycle_pools(self):
+        """Test the pool creation and destruction
+
+        Test Description:
+            Create a pool with all the capacity of all servers. Verify that the pool created
+            effectively used all the available storage. Destroy the pool and repeat these steps 100
```

Comment:
```suggestion
            effectively used all the available storage. Destroy the pool and repeat these steps 10
```
Typo? I see `range(10)`

Reply:
- [ ] fix typo line 144: 100 -> 10